allocation_manager/server/example.md

```diff
@@ -0,0 +1,53 @@
+# inside dev docker
+
+# compile your own version
+cd allocation_manager/server
+
+bazel build :allocation_manager && \
+
+../../bazel-bin/allocation_manager/server/allocation_manager_/allocation_manager --service_config=allocation_manager_config.textproto
+
+Send process to background by pressing ctrl-z followed by bg. Or start it in background with
+the `&` added to the end of command.
+
+Optionally, have example allocation code in `main`:
+
+```go
+	topo := apb.Topology{
+		Name: "Unit Name 2", Config: "Unit Config",
+	}
+	a := service.NewUnit(topo)
+
+	a.DoOperation("allocate")
```

Comment:
The RPC handlers in service.go use the same labels, but capitalized (i.e. Allocate). We should harmonize the labels; using both would cause confusion.

Reply:
I was looking if there is a way of getting the current function name as a string, but it does not seem very easy, although still doable. Afair, there were need for `reflection`